src/hyperion/external_interaction/callbacks/rotation/ispyb_callback.py

```diff
@@ -104,6 +104,12 @@ def activity_gated_start(self, doc: RunStart):
             self.uid_to_finalize_on = doc.get("uid")
         return super().activity_gated_start(doc)
 
+    def populate_axis_info_for_snapshot(
+        self, data_collection_info: DataCollectionInfo, omega_start: float | None
+    ):
+        # TODO in subsequent PR which depends on rotation snapshots 349
+        pass
+
```

Comment:
Should: I don't think we should do this at all for the rotation we can probably always just use the start angle given in the parameters. The snapshots for XRC and rotation have subtlety different purposes for the user:
* For XRC the snapshots are used work out what grids to do, therefore they will always be taken at the same angles as the datacollection
* For rotation they are just diagnostics to prove to the user we have centered on the crystal, they could be taken at arbitrary rotations that don't match the rotation we would be doing during data collection. So I think it'll potentially cause us issues in the future to tie the two together.

Reply:
This was going to be implemented for rotations before we had our previous discussion, however the PR for that never got raised as we decided not to do it. I will have removed the comment and leave it empty.

I was originally going to in the second PR for rotations, create a second snapshot event type that was rotation specific for this very reason, and then move the snapshot handlers into the callback subclasses and rename the event types, since they are plan-specific.

If you would like me to do that, I can move those changes across from the branch where I have saved them (minus the rotation snapshot event).